### ISSUE TITLE
Create a new personnal Homemaker style or modify an existing Homemaker style with Blender V 3.5

### ISSUE THREAD

**Issue by boes3d** [brunopostle/homemaker-addon#64]
Hello Mr Postle
Thank you for your updates regarding your latest versioning.
I would like with Blender V3.5 & BlenderBim V 230506 to create a new personnal style or modify an existing Arcade style, by example, with new dimensions like arch_600x600.
Is it possible ?
If so, can you give me some pointers?
Thanking you in advance.
Best regards.
Mr Rémy HAENEL

**Comment by brunopostle** [comment]
I'll do this, as I have not tried it with a recent blenderbim, but it will take some time

**Comment by brunopostle** [comment]
I've written a quick HOWTO, let me know if you have any questions:

https://github.com/brunopostle/homemaker-addon/wiki/Creating-a-Window-family

**Comment by brunopostle** [comment]
Yes, Homemaker is closely integrated with BlenderBIM and IFC, I don't see any other way of doing it.

BlenderBIM is a powerful tool, this school was modelled entirely as an IFC BIM using BlenderBIM (not by me):

![image_20230619093239](https://github.com/brunopostle/homemaker-addon/assets/1747315/6431c0ca-11a1-4b09-b716-8366afe67788)
![chateau-2_20230503170200](https://github.com/brunopostle/homemaker-addon/assets/1747315/f0fbb67c-f624-466c-9e09-ad819f764c36)

BlenderBIM has support for textured materials, but this functionality is only a few days old. I plan on adding support into Homemaker styles, but haven't started this. You can add textures to an existing IFC created by Homemaker and these will be preserved by BlenderBIM.
